cartography/intel/aws/ec2.py

```diff
@@ -1,6 +1,7 @@
 import botocore.config
 import logging
 import time
+import json
```

Comment:
Still used?  I might have missed it but I can't find any mention of it in the file.